README.md

```diff
@@ -98,44 +106,53 @@ Explicit steps for setting up CCM in a development environment.
     1. Note:  After saving, using "Paste JSON" under the "Method" menu, will show the corresponding JSON for the LTI Advantage configuration changes.  Enabling the LTI Advantage options adds them to the `scopes` key of the JSON.
 10. Copy the ID number of the LTI key created in Canvas.  The ID is the long number shown in the "Details" column of the "Developer Keys" page.  It usually looks like "`17700000000000nnn`".
 11. Click the "ON" part of the switch in the "State" column of your LTI key, so that it has a green background.
+
+##### Configure Canvas API key
+
 12. Go to Canvas "Developer Keys" management available from the "Admin" page for your account. Click the button for "+ Developer Key", then "+ API Key".
 13. Enter a name in the "Key Name" field, and under "Redirect URIs" enter:
-    `https://{ccm_app_hostname}/canvas/returnFromOAuth`
-    Where `{ccm_app_hostname}` is the ngrok hostname copied earlier (in step 2).
-    ***Note:*** Do ***NOT*** use the "Redirect URI (Legacy)" field.
+     `https://{ccm_app_hostname}/canvas/returnFromOAuth`
```

Comment:
So indentation is off according to your linter? Should be 5 spaces? Did you apply this change uniformly? I feel like I see 3 spaces some places, 4 most of the time, and then 5 in these places you added. Something doesn't seem right.

Reply:
According to the Markdown editor I use (https://typora.io/), yes.  Indentation depends on the number of digits in numbered list.  Single digit items need 4 spaces (room for one digit, a period, and two spaces), double digit items need 5 (room for two digits, a period, and two spaces).  Bullets need only 3 (room for the bullet and two spaces).  I think in Markdown additional lines are supposed to appear to be indented under the bullet/item that contains it.

For example…

* bullet 1
   info that
   goes with
   bullet 1
* bullet 2

9. single-digit item
    info that 
    goes with
    item 9
10. double-digit item
     info that
     goes with
     item 10